### PR TITLE
A quick first draft.

### PR DESCRIPTION
**Pull request by MikeiLL**

### CODE REVIEW

File: docs/trellis/master/database-access.md [hunk 1]
```diff
@@ -28,3 +28,6 @@ Accessing your databases with tools such as [Sequel Pro](https://www.sequelpro.c
 - Password: `example_dbpassword` (use the value of `db_password` from `group_vars/<environment>/vault.yml`)
 - SSH Host: `example.com`
 - SSH User: `web`
+
+NOTE: If you have provisioned your server to include local or remote ssh keys (see `group_vars/all/users.yml`), you can leave the SSH User password blank  in your DB Application, because SSH already knows about it. (In Sequel Pro the "key" icon toggles a browse field for SSH Key.) Additionally, [Trellis-cli](https://github.com/roots/trellis-cli) provides a one-liner to configure your connection via Sequel Pro or Table Plus.
```
Comment (swalkinshaw) [inline docs/trellis/master/database-access.md:32]: Didn't you add this in another PR before?

Comment (MikeiLL) [reply docs/trellis/master/database-access.md:32]: Oh, yes. Sorry about that. I think you had merged it and I hadn't updated my fork. Let me do that now...

Comment (MikeiLL) [reply docs/trellis/master/database-access.md:32]: Rebased origin/master. Do I create another PR now? I thought this one would update automatically? Sorry for the trouble.

Comment (swalkinshaw) [reply docs/trellis/master/database-access.md:32]: Another one might be easier. But yeah, forks don't automatically stay up to date. Honestly for small docs changes it's probably easier just to use GitHub's built in editor.

Comment (MikeiLL) [reply docs/trellis/master/database-access.md:32]: Thanks. I have been using the builtin web editor, but there's not a way to rebase through it, is there? I guess I could have deleted my fork, then re-forked before making the new changes. At the moment, I am having trouble sending a PR _without_ the previously merged codebase in it. Any suggestions?

**Pull request closed without merging**